tests/test_unit/test_utils/test_cdf.py

```diff
@@ -88,3 +89,37 @@ def test_find_node_type(filter_: dm.Filter | None, expected: dm.DirectRelationRe
 
     # Assert
     assert actual == expected
+
+
+def test_reduce_model() -> None:
```

Comment:
there seems to be no testing of exclusion by space, or that properties are excluded correctly